package.json

```diff
@@ -35,6 +35,7 @@
     "color-to-hsla": "^0.1.1"
   },
   "devDependencies": {
+    "cssfmt": "^2.1.5",
```

Comment:
Is there a specific reason why we are using this package? It looks like it was last published 8 years ago and has been superseded by [stylefmt}(https://www.npmjs.com/package/stylefmt), which hasn't been updated in 5 years. I'm just a bit concerned it is opting us into some vulnerabilities.

Reply:
I am open to using a different dependency.
I tried stylefmt and it had more vulnerabilities... I can fork and update deps to see if I can remedy the warnings.

Reply:
Would Prettier be an option? I'm not for or against it, but wanted to toss it out there as it seems to still be actively maintained.

Reply:
prettier was much more of a pain to add.

Reply:
That's kinda what I figured yeah, ugh

Reply:
fun fact! Prettier does not know how to behave like a cli!
`node ./cli.mjs | prettier > ./enhance.css --stdin-filepath ./enhance.css` does not work

There _may_ be a configuration that removes newlines from css files but there are **way** too many configuration options and I could not find it.

Reply:
I forked stylefmt and attempted to update the deps... error sends you to this upgrade guide:
https://evilmartians.com/chronicles/postcss-8-plugin-migration

Apparently it is a rewrite to update to postcss-8 so nope

Reply:
Should we just vendor `cssfmt` and apply patches as/when needed?

Reply:
As @macdonst pointed out it has vulnerabilities which may scare users. It is a development dependency and only formats one file but the high warning is not something we want to have in out npm output. Updating to deps with zero vulnerabilities breaks the package. It's a rewrite.

Reply:
Damn. Let me know if I can help!

Reply:
I’m going to apply all of the patches in the existing repo and see what’s left to fix once that’s done